src/applications/ivc-champva/10-10D/tests/e2e/fixtures/data/app-sd-cb-ohi.json

```diff
@@ -71,8 +71,6 @@
       "ssn": "123123123"
     },
     "certifierRole": "applicant",
-    "consentToMailMissingRequiredFiles": true,
-    "statementOfTruthSignature": "applicant jones",
-    "statementOfTruthCertified": true
+    "consentToMailMissingRequiredFiles": true
```

Comment:
E2E test automatically generates and inputs this data, so saving it in the JSON is unnecessary.